ios/RNSScreen.mm

```diff
@@ -977,6 +1026,87 @@ - (void)viewDidLayoutSubviews
   }
 }
 
+- (void)dismissViewControllerAnimated:(BOOL)flag completion:(void (^)())completion
+{
+  [super dismissViewControllerAnimated:flag completion:completion];
+  [self calculateHeaderHeightIsModal:self.screenView.isPresentedAsNativeModal];
+}
+
+// Checks whether this screen has any child view controllers of type RNScreensNavigationController.
+// Useful for checking if this screen has nested stack or is displayed at the top.
+- (BOOL)hasAnyChildNavigators
```

Comment:
I was also thinking about that name, but I'm wondering if there could be a situation where the screen would not be nested and it still could have RNSNavigationController 🤔

Reply:
There is no option for `RNSNavigationController` since it is the concept from our library for the stack navigator. If there was a child `RNSNavigationController` and it would not be a nested stack then it seems like a bug.

Reply:
Renamed in https://github.com/software-mansion/react-native-screens/pull/1802/commits/d2622c4f0b80ab3939c09a3c369c3403b8d36138.